src/azul/__init__.py

```diff
@@ -1141,6 +1141,14 @@ def cloudtrail_s3_bucket_region(self) -> str:
     def cloudtrail_trail_region(self) -> str:
         return self.environ['azul_cloudtrail_trail_region']
 
+    @property
+    def azul_notification_email(self) -> str:
+        return self.environ['AZUL_NOTIFICATION_EMAIL']
+
+    @property
+    def sns_topic_qualified_name(self):
```

Comment:
```suggestion
    def monitoring_topic_name(self):
```

You could have looked around for methods with a similar purpose and adopted their naming convention.